src/topology/discrete_quotient.lean

```diff
@@ -198,6 +200,27 @@ lemma of_le_proj_apply {A B : discrete_quotient X} (h : A ≤ B) (x : X) :
 
 end of_le
 
+/--
+When X is discrete, there is a `semilattice_inf_bot` instance on `discrete_quotient X`
+-/
+
+@[simp] lemma is_clopen_discrete [discrete_topology X] (x : set X) : is_clopen x :=
+  ⟨is_open_discrete _, is_closed_discrete _⟩
```

Comment:
This lemma should go after `is_closed_discrete` in `topology/order.lean`.

Reply:
Ok, thanks, I'll move it.

Reply:
I don't know if `is_clopen` is available yet there...

Reply:
Eric is probably right... If this is the case, I guess this lemma could go near where `is_clopen` is defined.

Reply:
Should I put it in `topology/subset_properties.lean` alongside other properties of `clopen` sets?

Reply:
Yes, I think that's a reasonable place for it.

Reply:
Ok, done.